src/navigation/pathnavigator.cpp

```diff
@@ -106,6 +106,15 @@ namespace {
         openspace::properties::Property::Visibility::User
     };
 
+    constexpr openspace::properties::Property::PropertyInfo JumpToFadeDurationInfo = {
+        "JumpToFadeDuration",
+        "JumpTo Fade Duration",
+        "The number of seconds the fading of the rendering should take per default when "
+        "jumping to a scene graph node using the 'openspace.pathnavigation.jumpTo' "
```

Comment:
Should we mention the script explicitly in here? I think naming the _feature_ instead would be a bit more understandable, since a normal user wouldn't interact with the script necessarily.  Also we might want to use the same value for other jump-like fadings, too

Reply:
Good point. Updated the description. 

Then, maybe we should already apply it to for example jumping to a lat long position on a globe? or a navigation state?

In that case, this property should probably be moved to the navigationhandler instead of the pathnavigator, to make it clear that it can be used for multiple types of transitions. Thoughts?